File: 2562.cpp
#include <iostream>
using namespace std;

int main() {
    int num, max = 0, a;
    for (int i = 0; i < 9; i++) {
        cin >> a;
        if (max < a) {
            max = a;
            num = i + 1;
        }
    }
    cout << max << '\n' << num;
}

File: 2562.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a[] = new int[9];
        int max = 0, num = 0;

        for (int i = 0; i < a.length; i++) {
            a[i] = sc.nextInt();
            if(max < a[i]) {
            max = a[i];
            num = i + 1;
            }
        }
        System.out.println(max);
        System.out.println(num);
    }
}
